pubsub/gcp/pubsub/pubsub.go

```diff
@@ -16,7 +17,9 @@ import (
 )
 
 const (
-	errorMessagePrefix                 = "gcp pubsub error:"
+	errorMessagePrefix = "gcp pubsub error:"
+
+	// metadata keys
```

Comment:
Please fix lint issues.

Reply:
@daixiang0 thank you for reviewing, I have just published a couple of commits to fix the lint issues and also add tests for the two new metadata properties.